### DISCUSSION TITLE
DAAN算法代码中MU的计算请教

### DISCUSSION THREAD

**Discussion by LIJiangkuan** [jindongwang/transferlearning#285] [Q&A] [answered]
王老师请问您的github里DAAN算法 transferlearning/code/deep/DAAN/train.py 中MU的计算公式MU = 1 - D_M/(D_M + D_C)貌似和您文章里的MU = D_M/(D_M + D_C)有矛盾，能麻烦你说下怎么回事吗，小白跪求回复。

**Comment by jindongwang** [comment, marked as answer]
我记得是paper写错了，忘记改了
________________________________
发件人: LIJiangkuan ***@***.***>
发送时间: Thursday, November 25, 2021 9:00:56 PM
收件人: jindongwang/transferlearning ***@***.***>
抄送: Subscribed ***@***.***>
主题: [jindongwang/transferlearning] DAAN算法代码中MU的计算请教 (Discussion #285)


王老师请问您的github里DAAN算法 transferlearning/code/deep/DAAN/train.py 中MU的计算公式MU = 1 - D_M/(D_M + D_C)貌似和您文章里的MU = D_M/(D_M + D_C)有矛盾，能麻烦你说下怎么回事吗，小白跪求回复。

―
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://github.com/jindongwang/transferlearning/discussions/285>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ABO7O7SRHXBMJWUBJPR7QC3UNYXQRANCNFSM5IYPMPRA>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Reply by LIJiangkuan** [reply]
收到，谢谢王老师，感谢王老师和您的团队提供这么好的资源库！

**Comment by miracle123ld** [comment]
王老师，请问一下迁移学习导论第二版的数据集怎么下载啊，按照步骤点了链接，不懂是不是步骤出差了，一直下不来

**Reply by jindongwang** [reply]
这个数据被很多人下载过了，没有问题。如果你下不了可能是你网络或者电脑有问题。我无法帮助你。
